src/i18n/strings/en_EN.json

```diff
@@ -1839,12 +1839,24 @@
         "files_button": "Files",
         "info": "Info",
         "pinned_messages": {
-            "empty": "Nothing pinned, yet",
-            "explainer": "If you have permissions, open the menu on any message and select <b>Pin</b> to stick them here.",
+            "empty_description": "Select a message and choose “Pin message” to include here.",
```

Comment:
This should use a string interpolation in case `Pin message` gets translated differently here than in the menu